src/components/DataTable/DataTable.tsx

```diff
@@ -146,6 +154,9 @@ class DataTable extends React.PureComponent<CombinedProps, DataTableState> {
       <div className={styles.Footer}>{footerContent}</div>
     ) : null;
 
+    const headerTotalsMarkup = !showTotalsInFooter ? totalsMarkup : null;
+    const footerTotalsMarkup = showTotalsInFooter ? totalsMarkup : null;
```

Comment:
```suggestion
    const footerTotalsMarkup = showTotalsInFooter ? (<tfoot>{totalsMarkup}</tfoot>) : null;
```

Reply:
Implemented! I put the <tfoot> in the actual render function to make the conditions there a bit more consistent and readable. Do you think that makes sense?

Reply:
Since the `tfoot` should only render when `footerTotalsMarkup` is true, I think it makes more sense to put it in line 158 instead. With your current implementation, the `tfoot` will always render, even when empty. We don't want that.